import.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+import csv
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String,DateTime,exists,Sequence
+from flask import Flask
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+
+
+# Import table definitions.
+
+app1 = Flask(__name__)
+
+# Tell Flask what SQLAlchemy databas to use.
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app1.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app1.app_context().push()
+# Link the Flask app1 with the database (no Flask app1 is actually being run yet).     
+
+db1 = SQLAlchemy()
+
+class Books(db1.Model):
```

Comment:
all the models must be in one file

Reply:
Hello Mam,
 They mentioned to write the table separately in assignment task on NGlms.
Hence I rewrote the code to create table.

Reply:
you need separate table, that can be written in the user table file itself

Reply:
hello mam, made the necessary changes. pls check. tq